tests/python-gpu/test_gpu_gdf.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+import pandas as pd
+import cudf.dataframe as gdf
```

Comment:
This test needs to be skipped if cudf is not installed. Our CI tests will try to run this and fail.

Reply:
Test made optional with a `try`, `except` python block on the import. If the import fails, it `print`s some informative messages, and will terminate the test by `return 0`.